app/models/sample.rb

```diff
@@ -20,6 +20,11 @@
 # - Heron: Heron samples get registered via the Api::V2::Heron::PlatesController
 # - Special samples: Samples such as {PhiX} are generated internally
 class Sample < ApplicationRecord # rubocop:todo Metrics/ClassLength
+  # See https://api.rubyonrails.org/classes/ActiveSupport/CurrentAttributes.html
+  class Current < ActiveSupport::CurrentAttributes
+    attribute :processing_manifest
+  end
+
```

Comment:
I considered setting a basic attribute on sample to handle this, but that would mean doing so for new samples, as well as found ones. Plus, if we ever reloaded the samples, the attribute would get blitzed.